mtest/libmscore/compat206/articulations-double-ref.mscx

```diff
@@ -5,15 +5,6 @@
     <currentLayer>0</currentLayer>
     <Division>480</Division>
     <Style>
-      <pageWidth>8.26771</pageWidth>
```

Comment:
You'll see a lot of this below here.  It is an improvement.  These values are all rounded versions of the baseStyle values.  The PR's code recognizes this and sets them to the default values internally.  Thus the values are optimized out when you save the file.